docs/extend/i18n.md

```diff
@@ -19,7 +25,7 @@ As a rule, a Flarum site can only display translations corresponding to the lang
 3. As a last-ditch effort, it will look for a "generic" English translation of the output.
 4. If none of the above are available, it will give up and display a translation key.
 
-Since English translations could be the only thing standing between forum users and unsightly translation keys, we strongly recommend including a complete set of English resources with every extension. (You will also need to include English resources if you wish to list your extension on the Flarum Marketplace.)
+Since English translations could be the only thing standing between forum users and unsightly translation keys, we strongly recommend including a complete set of English resources with every extension. (You will also need to include English resources if you wish to list your extension on the Flarum Marketplace, when that launches). Generally, extension developers will include english translations in their extension, and other language translations in a [language pack](language-packs.md).
```

Comment:
```suggestion
Since English translations could be the only thing standing between forum users and unsightly translation keys, we strongly recommend including a complete set of English resources with every extension. (You will also need to include English resources if you wish to list your extension on the Flarum Marketplace, when that launches). Generally, extension developers will include English translations in their extension, and other language translations in a [language pack](language-packs.md).
```